src/test/ui/macros/missing-comma.stderr

```diff
@@ -7,32 +7,68 @@ LL |     println!("{}" a);
 error: no rules expected the token `b`
   --> $DIR/missing-comma.rs:22:12
    |
-LL |     foo!(a b);
-   |           -^
-   |           |
-   |           help: missing comma here
+LL | / macro_rules! foo {
+LL | |     ($a:ident) => ();
+LL | |     ($a:ident, $b:ident) => ();
+LL | |     ($a:ident, $b:ident, $c:ident) => ();
+LL | |     ($a:ident, $b:ident, $c:ident, $d:ident) => ();
+LL | |     ($a:ident, $b:ident, $c:ident, $d:ident, $e:ident) => ();
+LL | | }
+   | |_- when calling this macro
+...
+LL |       foo!(a b);
+   |             -^ no rules expected the token `b`
+   |             |
+   |             help: missing comma here
 
 error: no rules expected the token `e`
   --> $DIR/missing-comma.rs:24:21
    |
-LL |     foo!(a, b, c, d e);
-   |                    -^
-   |                    |
-   |                    help: missing comma here
+LL | / macro_rules! foo {
+LL | |     ($a:ident) => ();
+LL | |     ($a:ident, $b:ident) => ();
+LL | |     ($a:ident, $b:ident, $c:ident) => ();
+LL | |     ($a:ident, $b:ident, $c:ident, $d:ident) => ();
+LL | |     ($a:ident, $b:ident, $c:ident, $d:ident, $e:ident) => ();
+LL | | }
+   | |_- when calling this macro
```

Comment:
do we want to highlight the *whole* definition? Might we be better off just highlighting the `macro_rules! foo` portion?

Reply:
As is it will only ever show 7 lines, at most of the macro. For the simple case that's enough to spot the problem without having to dig for the source. Having said that, I can change the PR to only point at the `macro_rules! foo` portion if needed.

Reply:
Done.